native/src/components/__tests__/Failure.spec.tsx

```diff
@@ -7,27 +7,27 @@ import buildConfig from '../../constants/buildConfig'
 import Failure from '../Failure'
 
 jest.mock('react-i18next')
+jest.mock('styled-components', () => ({
```

Comment:
Can't you just use `wrapWithTheme` instead of mocking now? This is okay for me as well though.